zwave_js_server/client.py

```diff
@@ -386,3 +247,14 @@ def _check_server_version(self, server_version: str) -> None:
                 "Connected to a Zwave JS Server with an untested version, \
```

Comment:
Should we really warn for this? We're going to reach this very quick. I don't think we should.

Reply:
Not sure. We can think about it and address later. :+1:

Reply:
Maybe only when the minor version changes ? Not when the patch version is different.

Reply:
Or simply do not warn. We have the minimal server version constant if we have breaking api changes

Reply:
We should only see if major version is higher and then warn. Rest should be safe if we follow semver